data/flux_influxdb_minor_versions.yml

```diff
@@ -0,0 +1,24 @@
+# This is a list of all InfluxDB minor versions that support Flux.
+# It is used by flux-build-scripts/update-flux-versions.js to generate the
+# "Supported Versions" section of the Flux page.
+# When new versions of InfluxDB are released that support Flux, this list should
```

Comment:
Just an idea: it would be cool to have a CI checklist prompt (if we don't ignore it) whenever we add a version or product.